deal.II-toolchain/packages/scalapack.package

```diff
@@ -0,0 +1,86 @@
+################################################################################
+## ScaLAPACK                                                                  ##
+################################################################################
+
+VERSION=2.1.0
+EXTRACTSTO=scalapack-${VERSION}
+
+NAME=v${VERSION}
+PACKING=.tar.gz
+
+SOURCE=https://github.com/Reference-ScaLAPACK/scalapack/archive/
+CHECKSUM=e9ae970238030fa1e8ee022a9d3e6fbb
+
+BUILDCHAIN=cmake
+
+BUILDDIR=${BUILD_PATH}/scalapack-${VERSION}
+INSTALL_PATH=${INSTALL_PATH}/scalapack-${VERSION}
+
+
+
+# Set compilers & compiler options
+if [ ! -z "${CC}" ]; then
+    CONFOPTS="${CONFOPTS} \
+      -D CMAKE_C_COMPILER=${CC}"
+fi
+
+if [ ! -z "${CXX}" ]; then
+    CONFOPTS="${CONFOPTS} \
+      -D CMAKE_CXX_COMPILER=${CXX}"
+fi
+
+if [ ! -z "${FC}" ]; then
+    CONFOPTS="${CONFOPTS} \
+      -D CMAKE_Fortran_COMPILER=${FC}"
+fi
+
+# Set Fortran flags
+FORTRAN_MAJOR_VERSION=$(echo __GNUC__ | ${FC} -E -xc - | tail -n 1)
+if [ ${FORTRAN_MAJOR_VERSION} > 9 ]; then
+  CONFOPTS="${CONFOPTS} \
+    -D CMAKE_Fortran_FLAGS='-g -O3 -fallow-argument-mismatch' "
+else
+  CONFOPTS="${CONFOPTS} \
+    -D CMAKE_Fortran_FLAGS='-g -O3' "
+fi
+
+# Set C flags
+CONFOPTS="${CONFOPTS} \
+  -D CMAKE_C_FLAGS:STRING='-fPIC -g -O3 ' \
+  -D CMAKE_POSITION_INDEPENDENT_CODE:BOOL=ON"
```

Comment:
is this needed? `fPIC` should be enough, or?

Reply:
Without this additional -fPIC I got an error while building Trilinos, that I need to recompile the library using -fPIC.

Reply:
fpic is good, but the second statement for independent code seems to be not needed.

Reply:
I double checked it, it seems like that line is necessary.
Without `-D CMAKE_POSITION_INDEPENDENT_CODE:BOOL=ON` Trilinos fails to build.